coralproject/talk
# [CORL-493] Featured Comment Notifications

## What does this PR do?

Adds featured comment notification support.

## How do I test this PR?

1. Enable the notification type under your profile settings.
2. Write a comment.
3. With an admin user, feature the comment.
4. Check your email!
